<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MacBooks-MacBook-Pro.local" failures="1" tests="13" name="TS03_Malls" time="94.007" errors="0" timestamp="14 Jun 2020 11:19:04 GMT" skipped="0">
  <testcase classname="TS03_Malls" name="isFifthMallLocationCorrect" time="8.185"/>
  <testcase classname="TS03_Malls" name="isFourthMallWorkingHoursCorrect" time="2.986"/>
  <testcase classname="TS03_Malls" name="isFifthMallWorkingHoursCorrect" time="2.986"/>
  <testcase classname="TS03_Malls" name="areAllImagesDisplayed" time="4.276"/>
  <testcase classname="TS03_Malls" name="isFourthMallLocationCorrect" time="6.466"/>
  <testcase classname="TS03_Malls" name="isFirstMallLocationCorrect" time="7.196">
    <failure message="expected [Al Nakheel Mall] but found [Exit 9]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Al Nakheel Mall] but found [Exit 9]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at TS03_Malls.isFirstMallLocationCorrect(TS03_Malls.java:45)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
]]>
    </failure>
  </testcase> <!-- isFirstMallLocationCorrect -->
  <testcase classname="TS03_Malls" name="isMallsTabDisplayed" time="2.648"/>
  <testcase classname="TS03_Malls" name="isThirdMallLocationCorrect" time="5.249"/>
  <testcase classname="TS03_Malls" name="isSecondMallLocationCorrect" time="5.182"/>
  <testcase classname="TS03_Malls" name="isThirdMallWorkingHoursCorrect" time="9.243"/>
  <testcase classname="TS03_Malls" name="isFirstMallWorkingHoursCorrect" time="2.890"/>
  <testcase classname="TS03_Malls" name="isSecondMallWorkingHoursCorrect" time="8.473"/>
  <testcase classname="TS03_Malls" name="isSuccessfullyRedirectedToGoogleMaps" time="28.227"/>
</testsuite> <!-- TS03_Malls -->
